### DISCUSSION TITLE
Importing grooves

### DISCUSSION THREAD

**Discussion by ShoveD01** [bangcorrupt/hacktribe#266] [Q&A] [answered]
Is there any info on how to import grooves, can't seem to find it anywhere?

**Comment by bangcorrupt** [comment, marked as answer]
So far only the `main` function of [e2_groove_template.py](https://github.com/bangcorrupt/hacktribe/blob/main/scripts/e2_groove_template.py).  I've started work adding groove and scale editing to `ht-cli`, but it might not be available for a while.
